prime-dotnet-webapi/ViewModels/AutoMapping.cs

```diff
@@ -74,5 +76,21 @@ public AutoMapping()
         CreateMap<PlrProvider, PlrProvider>()
             .ForMember(dest => dest.Id, opt => opt.Ignore())
             .ForMember(dest => dest.Ipc, opt => opt.Ignore());
+
+        CreateMap<AuthorizedUser, AuthorizedUserViewModel>()
+            .ForMember(dest => dest.UserId, opt => opt.MapFrom(src => src.Party.UserId))
+            .ForMember(dest => dest.FirstName, opt => opt.MapFrom(src => src.Party.FirstName))
+            .ForMember(dest => dest.LastName, opt => opt.MapFrom(src => src.Party.LastName))
+            .ForMember(dest => dest.GivenNames, opt => opt.MapFrom(src => src.Party.GivenNames))
+            .ForMember(dest => dest.DateOfBirth, opt => opt.MapFrom(src => src.Party.DateOfBirth))
+            .ForMember(dest => dest.PreferredFirstName, opt => opt.MapFrom(src => src.Party.PreferredFirstName))
+            .ForMember(dest => dest.PreferredMiddleName, opt => opt.MapFrom(src => src.Party.PreferredMiddleName))
+            .ForMember(dest => dest.PreferredLastName, opt => opt.MapFrom(src => src.Party.PreferredLastName))
+            .ForMember(dest => dest.VerifiedAddress, opt => opt.MapFrom(src => src.Party.VerifiedAddress))
+            .ForMember(dest => dest.PhysicalAddress, opt => opt.MapFrom(src => src.Party.PhysicalAddress))
+            .ForMember(dest => dest.Email, opt => opt.MapFrom(src => src.Party.Email))
+            .ForMember(dest => dest.Phone, opt => opt.MapFrom(src => src.Party.Phone))
+            .ForMember(dest => dest.SmsPhone, opt => opt.MapFrom(src => src.Party.SMSPhone))
+            .ForMember(dest => dest.JobRoleTitle, opt => opt.MapFrom(src => src.Party.JobRoleTitle));
```

Comment:
Just in case you weren't aware, the default way to do this kind of mapping is to use naming conventions. AutoMapper will map automatically map src.Party.FirstName to dest.PartyFirstName (for example).

_However_, we want to name our viewmodels like it's a flat model. So I did some digging and I found the proper way to do this:

https://docs.automapper.org/en/stable/Flattening.html#includemembers

Short of it is we make a mapping from Party to AuthorizedUserViewModel (so it knows to match by property names), then we invoke .IncludeMembers() to "hoist up" the child objects and also map them to the destination object:

```C#
CreateMap<AuthorizedUser, AuthorizedUserViewModel>().IncludeMembers(src => src.Party);
CreateMap<Party, AuthorizedUserViewModel>();
```

Reply:
@james-hollinger sweetness!